provider/README.md

```diff
@@ -102,19 +102,19 @@ Data: `{ "trips": [] }`, an array of objects with the following structure
 
 ### Vehicle Types
 
-| vehicle_type | description |
-|-------|-------------|
-| bicycle | |
-| scooter | |
-| recumbent | |
+| `vehicle_type` |
+|--------------|
+| bicycle      |
+| scooter      |
+| adaptive     |
```

Comment:
I like replacing recumbent with adaptive. @LADOTBikeshare can you thing of any other vehicle types we should start supporting?

Reply:
Technically, Adaptive bikes come in many shapes and sizes based on the type of injury they are accommodating.  I think we need to stay as specific as possible.

Reply:
It might be useful to add that `Description` column back and populate with some clear distinctions, regardless of what keyword we choose.

Reply:
Another option is to just support `bicycle` and `scooter` for now, until companies deploy additional vehicle types at which point they can submit a PR 😄

Reply:
@thekaveman Let's go with the latter option then.

Reply:
2a98c22